### PR TITLE
Phase2-geox02 Make workflow for a version with HGCal V13

### PR DESCRIPTION
**Pull request by bsunanda**
#### PR description:

Make workflow (2026D59) for a version with HGCal V13. But the new workflow needs merging of 2 PR's (#30423 and #30428). Also corrects scenarios for 2026D57 and 2026D58 where two files intended to add similar (but not the same) volumes are added

#### PR validation:. 

Tested along with the 2 PR's

#### if this PR is a backport please specify the original PR and why you need to backport that PR:

Nothing special

### CODE REVIEW

**Comment by bsunanda** [comment]
@cmsbuild Please test

File: Configuration/Geometry/python/dict2026Geometry.py [hunk 1]
```diff
@@ -1277,6 +1420,31 @@
...
+        ],
+        4 : [
+            'Geometry/ForwardSimData/data/zdcProdCuts.xml',
+            'Geometry/ForwardSimData/data/ForwardShieldProdCuts.xml',
+        ],
+        "reco" :[
+            'from Geometry.ForwardGeometry.ForwardGeometry_cfi import *',
+        ]
+    },
     "F5" : {
```
Comment (rovere) [inline Configuration/Geometry/python/dict2026Geometry.py:1448]: Are you sure you want to redefined "F4" key?

File: Configuration/PyReleaseValidation/python/upgradeWorkflowComponents.py [hunk 2]
```diff
@@ -852,6 +854,13 @@ def condition(self, fragment, stepList, key, hasHarvest):
         'Era' : 'Phase2C12',
         'ScenToRun' : ['GenSimHLBeamSpotFull','DigiFullTrigger','RecoFullGlobal', 'HARVESTFullGlobal'],
     },
+    '2026D59' : {
+        'Geom' : 'Extended2026D59',
+        'HLTmenu': '@fake2',
+        'GT' : 'auto:phase2_realistic_T15',
+        'Era' : 'Phase2C11',
```
Comment (rovere) [inline Configuration/PyReleaseValidation/python/upgradeWorkflowComponents.py:861]: Isn't it misleading to couple `Phase2C11` with `C13` geometry?

**Comment by rovere** [comment]
I think this PR lucks a clear explanation of what it introduces as a scenario in the usual `Configuration/Geometry/README.md` file. Especially taking into account that it touches more than pure `HGCAL` components.

File: Configuration/PyReleaseValidation/python/upgradeWorkflowComponents.py (hunk 2 above)
Comment (bsunanda) [reply Configuration/PyReleaseValidation/python/upgradeWorkflowComponents.py:861]: We have only defined Phase2C11 which will work for V11, V12, V13 geometries of HGCal like Phase2C9 which works for V9 and V10

File: Configuration/Geometry/python/dict2026Geometry.py (hunk 1 above)
Comment (bsunanda) [reply Configuration/Geometry/python/dict2026Geometry.py:1448]: Thanks for pointing this out. F4 is copied and pasted twice. I have removed the second one

**Comment by bsunanda** [comment]
@cmsbuild Please test

**Comment by kpedro88** [comment]
@bsunanda there is a conflict: #30340 already claimed D59 and D60 for ETL geometry updates. However, their workflows currently aren't functioning. @icosivi @fabiocos please advise how you want to proceed.

**Comment by cvuosalo** [comment]
Looks good so far. I'll wait to approve until the issue with #30340 is resolved.

**Comment by bsunanda** [comment]
@icosivi @kpedro88 @cvuosalo @fabiocos Please let me know how to proceed. If Fabio's PR gets a precedence I would simply close this PR and take out the ones needed for HGCal urgently in a separate PR

**Comment by bsunanda** [comment]
It will be nice to know the solution quickly. I am ready with either process (to proceed with this PR or wait for MTD PR to converge first)

**Comment by fabiocos** [comment]
@bsunanda @kpedro88 I would suggest this PR moves first, we are still debugging with @icosivi and @casarsa the ETLDetId issue, the rest may be casted in any scenario afterwards just moving the ID, no reason to keep this on hold if it is ready to go.

**Comment by kpedro88** [comment]
please test workflow 27834.0
(just to make sure)

**Comment by bsunanda** [comment]
@kpedro88 @silviodonato Please not that this needs merging of 2 PR's (#30423 and #30428). Out of these 2, #30423 cannot be merged now because of conflict for reverting back Run4-hgx247 (#30323)

**Comment by kpedro88** [comment]
@bsunanda okay, today let's get everything fixed up for the TDR (I think all the pieces are available now), and then we'll retest this for HGCal tomorrow

**Comment by bsunanda** [comment]
@cmsbuild Please test

**Comment by bsunanda** [comment]
@kpedro88 Could you start testing 27834.0 with #30423

**Comment by kpedro88** [comment]
please test workflow 27834.0 with #30423

**Comment by kpedro88** [comment]
+upgrade

**Comment by chayanit** [comment]
+1

**Comment by bsunanda** [comment]
@cvuosalo @ianna Can you approve this PR

**Comment by cvuosalo** [comment]
+1

**Comment by bsunanda** [comment]
@franzoni Could you approve this PR?

**Comment by kpedro88** [comment]
@silviodonato please approve this so I can rebase #30477

**Comment by silviodonato** [comment]
+1

**Pull request merged**